src/Api/SecretsManager/Models/Request/GetSecretsRequestModel.cs

```diff
@@ -1,9 +1,24 @@
 ﻿using System.ComponentModel.DataAnnotations;
+using System.Diagnostics;
 
 namespace Bit.Api.SecretsManager.Models.Request;
 
-public class GetSecretsRequestModel
+public class GetSecretsRequestModel : IValidatableObject
 {
     [Required]
     public IEnumerable<Guid> Ids { get; set; }
+    public IEnumerable<ValidationResult> Validate(ValidationContext validationContext)
+    {
+        bool isDistinct = Ids.Distinct().Count() == Ids.Count();
```

Comment:
```suggestion
        var isDistinct = Ids.Distinct().Count() == Ids.Count();
```

The project's coding style prefers us to use `var` here 
https://github.com/bitwarden/server/blob/a1d609b208f37f45bd8cf8d7c22769cd0455b281/.editorconfig#L88-L93